packages/typir-langium/README.md

```diff
@@ -0,0 +1,21 @@
+# Typir integration for Langium
+
+Engineering types beside using Langium.
```

Comment:
Suggestion:

Typir-Langium is a framework for type checking of languages developed with [Langium](https://langium.org/).